### PR TITLE
Fixing #779 by settings http protocol and allowing to set keep alive …

### PR DESCRIPTION
**Pull request by bloodbare**
…timeout

### CODE REVIEW

File: guillotina/_settings.py [hunk 1]
```diff
@@ -91,5 +91,7 @@
     "search_parser": "default",
     "object_reader": "guillotina.db.reader.reader",
     "thread_pool_workers": 32,
+    "keep_alive": 5,
```
Comment (vangheem) [inline guillotina/_settings.py]: 5 is very low and I'm not sure it'll actually help the situation if this is the same problem that I've seen.

fwiw, nginx defaults to 75sec keep alive I believe.

File: guillotina/_settings.py [hunk 2]
```diff
@@ -91,5 +91,7 @@
...
+    "http_protocol": "h11"
```
Comment (vangheem) [inline guillotina/_settings.py]: it's a shame we end up with h11 here

**Comment by masipcat** [comment]
I'm not sure if we want to expose all these options in the cli / app_settings in the top level. I'd suggest something like:
```python
app_settings = {
    "server_settings": {
         "uvicorn": { ... }
     }
}
```
that is mapped to the uvicorn Config

File: guillotina/_settings.py (hunk 1 above)
Comment (bloodbare) [reply guillotina/_settings.py]: default keep alive timeout at uvicorn is 5 (https://github.com/encode/uvicorn/blob/master/uvicorn/config.py#L134)

File: guillotina/_settings.py (hunk 2 above)
Comment (bloodbare) [reply guillotina/_settings.py]: Its temporal until encode/uvicorn#344 is fixed

**Pull request merged**